src/spaces/vectorspaces.jl

```diff
@@ -172,32 +177,35 @@ Base.conj(V::ElementarySpace{ℝ}) = V
 """
     abstract type InnerProductSpace{𝕜} <: ElementarySpace{𝕜} end
 
-Abstract type for denoting vector with an inner product and a corresponding metric, which
-can be used to raise or lower indices of tensors.
+Abstract type for denoting vector with an inner product, thus a canonical mapping from 
```

Comment:
This should have been: `... for denoting vector spaces with an inner product...`, i.e. i had forgotten the word spaces.

Reply:
Done!